pkg/install/squash.go

```diff
@@ -124,7 +120,7 @@ func InstallSquash(cs *kubernetes.Clientset, namespace, containerRepo, container
 					Containers: []v1.Container{
 						{
 							Name:  SquashName,
-							Image: fmt.Sprintf("%v/%v:%v", containerRepo, containerVersion, version.SquashImageTag),
+							Image: fmt.Sprintf("%v/%v:%v", containerRepo, SquashName, version.SquashImageTag),
```

Comment:
This correctly fixes the issue, however we would still like to be able to overwrite the tag from the command line flag, as in https://github.com/solo-io/squash/pull/139
I'm sorry, I did not see your PR before I made the fix, otherwise I would have merged yours

Reply:
Never mind. It is also a pleasure to see my issue resolved quickly :)